### ISSUE TITLE
About the training epochs

### ISSUE THREAD

**Issue by zhengyb** [MaybeShewill-CV/lanenet-lane-detection#600]
请教一下，在Readme.md中，“In my experiment the training epochs are 80010, batch size is 4, initialized learning rate is 0.001 and use polynomial decay with power 0.9.” 这个经验值对最新版本的训练代码还是有效的吗？我在查看issues的时候看到下面这个关于epoch的解释，但我看了最新版本的训练代码，我理解一次epoch就是迭代完所有的数据。是我的理解有误吗？

我的环境：Tensorflow=1.15.5

> @FlyingSharp 你看下代码就知道了啊 这里的epoch一次就是一个batch的数据, 不是迭代完所有的数据算作一个epoch.这个看代码应该很清楚了吧. 如果你说这个不叫一个epoch的话 那也算你对 就叫一次iter吧：） 

 _Originally posted by @MaybeShewill-CV in [#368](https://github.com/MaybeShewill-CV/lanenet-lane-detection/issues/368#issuecomment-607649042)_

**Comment by MaybeShewill-CV** [comment]
@zhengyb 一个epoch迭代完所有的数据
